server/index.ts

```diff
@@ -54,24 +52,29 @@ try {
 } catch (e) {
 	throw new Error("We require Node.js version 10 or later; you're using " + process.version);
 }
+
 try {
 	require.resolve('../.sim-dist/index');
 } catch (e) {
 	throw new Error("Dependencies are unmet; run `node build` before launching Pokemon Showdown again.");
 }
 
-const FS = require('../.lib-dist/fs').FS;
+import {FS} from '../lib/fs';
 
 /*********************************************************
  * Load configuration
  *********************************************************/
 
-const ConfigLoader = require('../.server-dist/config-loader');
+// global becomes much easier to use if declared as an object
+declare const global: any;
+
+import * as ConfigLoader from './config-loader';
 global.Config = ConfigLoader.Config;
 
-global.Monitor = require('../.server-dist/monitor').Monitor;
+import { Monitor } from './monitor';
```

Comment:
```suggestion
import {Monitor} from './monitor';
```

Could you grep `import { ` (with the space) and fix those, too?

Reply:
Resolved this in the latest commit